### PR TITLE
feat: neurons new wizard modal

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Use new wizard modal design in neurons related modals

# Changes

- use new modal
- create `TransactionSource` to display the source on stake neuron as in transaction
- fix missing back in delay confirm
- `CreateNeuronModal` renamed in `StakeNeuronModal`

# Screenshots

Capture d’écran 2022-10-12 à 09 35 31
Capture d’écran 2022-10-12 à 09 35 39
Capture d’écran 2022-10-12 à 09 56 15
Capture d’écran 2022-10-12 à 09 56 18
Capture d’écran 2022-10-12 à 09 56 21
Capture d’écran 2022-10-12 à 09 56 24
Capture d’écran 2022-10-12 à 10 15 06
Capture d’écran 2022-10-12 à 10 15 44
Capture d’écran 2022-10-12 à 10 37 41
Capture d’écran 2022-10-12 à 10 37 44
Capture d’écran 2022-10-12 à 10 37 48
Capture d’écran 2022-10-12 à 10 37 51
Capture d’écran 2022-10-12 à 10 38 16
Capture d’écran 2022-10-12 à 10 38 20
Capture d’écran 2022-10-12 à 10 47 31
Capture d’écran 2022-10-12 à 10 47 33
Capture d’écran 2022-10-12 à 10 47 36
Capture d’écran 2022-10-12 à 10 47 38
Capture d’écran 2022-10-12 à 10 47 51
Capture d’écran 2022-10-12 à 10 47 54
Capture d’écran 2022-10-12 à 10 47 58
Capture d’écran 2022-10-12 à 10 48 01

### CODE REVIEW

File: frontend/src/lib/components/neurons/NnsNeuronsFooter.svelte [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
   import Footer from "$lib/components/common/Footer.svelte";
   import { Toolbar } from "@dfinity/gix-components";
-  import CreateNeuronModal from "$lib/modals/neurons/CreateNeuronModal.svelte";
+  import CreateNeuronModal from "$lib/modals/neurons/StakeNeuronModal.svelte";
```
Comment (lmuntaner) [inline frontend/src/lib/components/neurons/NnsNeuronsFooter.svelte:4]: Worth renaming also `CreateNeuronModal` when importing?

**Review by lmuntaner — APPROVED** [review]
Looking very good!!

Minor comment on renaming a component when imported. But already approving.

**Comment by lmuntaner** [comment]
Btw, can you remove the LegacyModal already? Or is it still being used?

File: frontend/src/lib/components/neurons/NnsNeuronsFooter.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/neurons/NnsNeuronsFooter.svelte]: Definitely

Comment (peterpeterparker) [reply frontend/src/lib/components/neurons/NnsNeuronsFooter.svelte]: Thx

**Comment by peterpeterparker** [comment]
> Btw, can you remove the LegacyModal already? Or is it still being used?

Not yet. Canisters.

**Pull request merged**